src/main/java/com/google/api/codegen/py/PythonGapicContext.java

```diff
@@ -253,14 +253,17 @@ private String returnTypeComment(
     return Splitter.on("\n").splitToList(s);
   }
 
-  /** Generate comments lines for a given method's description. */
-  public List<String> methodDescriptionComments(Method method) {
+  private String getTrimmedDocs(ProtoElement elt) {
     String description = "";
-    if (method.hasAttribute(ElementDocumentationAttribute.KEY)) {
-      String sphinxified = getSphinxifiedScopedDescription(method);
-      description = sphinxified.replaceAll("\\s*\\n\\s*", "\n");
+    if (elt.hasAttribute(ElementDocumentationAttribute.KEY)) {
+      description = getSphinxifiedScopedDescription(elt);
     }
-    return splitToLines(description);
+    return description.replaceAll("\\s*\\n\\s*", "\n");
```

Comment:
This changes the logic a bit to move this outside the `if` test. Is that a deliberate bug fix?